packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -95,21 +103,28 @@ export function listenForSyncEvent(actions, store) {
           // Tell the user something is wrong with the key state on
           // the server and the key needs to be recreated
           notif = {
-            title: 'Actual has updated the syncing format',
+            title: i18n.t('Actual has updated the syncing format', {}),
             message:
-              'This happens rarely (if ever again). The internal syncing format ' +
-              'has changed and you need to reset sync. This will upload data from ' +
-              'this device and revert all other devices. ' +
-              '[Learn more about what this means](https://actualbudget.org/docs/getting-started/sync/#what-does-resetting-sync-mean).' +
-              '\n\nOld encryption keys are not migrated. If using ' +
-              'encryption, [reset encryption here](#makeKey).',
+              i18n.t(
+                'This happens rarely (if ever again). The internal syncing format ' +
+                  'has changed and you need to reset sync. This will upload data from ' +
+                  'this device and revert all other devices.',
+              ) +
+              ' ' +
+              i18n.t(
+                '[Learn more about what this means](https://actualbudget.org/docs/getting-started/sync/#what-does-resetting-sync-mean).',
+              ) +
+              '\n\n' +
+              i18n.t(
+                'Old encryption keys are not migrated. If using encryption, [reset encryption here](#makeKey).',
+              ),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think this entire block should be a single translation. Rather than splitting in 3x pieces.